File: src/main/java/com/bonvio/mvc/ServiceController.java
package com.bonvio.mvc;

import com.bonvio.model.People;
import com.bonvio.service.PeopleService;
import java.util.List;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpStatus;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.*;


@Controller
@RequestMapping(value = "service")
public class ServiceController {
    
    @Autowired
    PeopleService peopleService;
    
    @RequestMapping(value = "get" ,method = RequestMethod.GET)
    @ResponseBody
    public Object getPeople() {
        return peopleService.get();
    }
    
    @RequestMapping(value = "getSites/{id}", method = RequestMethod.GET)
    @ResponseBody
    public Object getSites(@PathVariable ("id") long people_id) {
        return peopleService.getSite(people_id);
    }

    @RequestMapping(method = RequestMethod.PUT)
    @ResponseStatus(HttpStatus.CREATED)
    public void addPeople(@RequestBody People people) {
        peopleService.add(people);
    }

    @RequestMapping(value="/{id}", method = RequestMethod.DELETE)
    @ResponseStatus(HttpStatus.ACCEPTED)
    public void deletePeople(@PathVariable ("id") long people_id) {
        People people = new People();
        people.setPeople_id(people_id);
        peopleService.delete(people);
    }

}

File: src/main/webapp/resources/js/serviceController.js
app.controller('ServiceController', function ($scope, $http) {
    
    $http.get('service/get').then(function (data) {
        $scope.peoples = data.data;
    });
    
    $scope.getSite = function (id) {
        $http.get('service/getSites/'+ $scope.peoples[id].people_id).then(function (data) {
            $scope.sites = data.data;
            console.log($scope.sites);
        });
    };
    
    $scope.savePeople = function () {
        $http.put('service', $scope.addPeople).then(function (data) {
            $scope.addPeople = {};
            $http.get('service/get').then(function (data) {
                $scope.peoples = data.data;
            });
        });
    };
    
    $scope.delPeople = function (id) {
        $http.delete('service/' + $scope.peoples[id].id).then(function (data) {
            $http.get('service/get').then(function (data) {
                $scope.peoples = data.data;
            });
        });
    };
});

